src/AWS/Decode.elm

```diff
@@ -29,3 +29,51 @@ responseWrapperDecoder actionName dataName dataDecoder =
                         |> JDP.required "RequestId" JD.string
                     )
             )
+
+
+tryFields : List String -> JD.Decoder a -> JD.Decoder (Maybe JD.Value)
+tryFields fields decoder =
+    fields
+        |> List.map
+            (\field ->
+                JD.field field (JD.maybe JD.value)
+            )
+        |> JD.oneOf
```

Comment:
doing `oneOf` was not enough, as it would still silently fail if the member was found, but decoding failed